starter/source/interfaces/inter3d1/i11remlin.F

```diff
@@ -279,13 +280,14 @@ SUBROUTINE I11REMLINE(
               ENDIF
             ENDDO
             KREMNODE(I+1) = KREMNODE(I) + CPT1
-          ELSEIF (IGAP == 1) THEN 
+          ELSE 
```

Comment:
here you also change the behavior for Iremgap=2 + Igap = 2 , is it the behavior you expect ?

Reply:
Done